src/webui/static/app/tvheadend.js

```diff
@@ -489,7 +489,7 @@ function accessUpdate(o) {
         var chepg = new Ext.TabPanel({
             activeTab: 0,
             autoScroll: true,
-            title: _('Channel / EPG'),
+            title: _('Channel list'),
             iconCls: 'television',
```

Comment:
If this is the config tab, doesn't it still cover both channels and EPG settings? Or am I on an old version?